scripts/compose/graphite/storage/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Alternatively, would it make sense to put the `.gitignore` into the parent directory and ignore the `storage` sub-directory?

Reply:
No, then the user would have to manually create the `storage` directory, otherwise the Graphite service would fail on start.